lib/AST/Module.cpp

```diff
@@ -2573,6 +2584,18 @@ bool SourceFile::isImportedAsSPI(const ValueDecl *targetDecl) const {
   return false;
 }
 
+bool ModuleDecl::isImportedAsPackage(const ValueDecl *targetDecl) const {
+//  FIXME
+//  auto targetModule = targetDecl->getModuleContext();
+//  for (auto &import : *Imports) {
+//    if (import.options.contains(ImportFlags::PackageAccessControl) &&
+//        targetModule == import.module.importedModule) {
+//        return true;
+//    }
+//  }
+  return false;
+}
```

Comment:
With all the type-checking done at the file-level, I don't think we need this function at the module level. The SPI equivalent was apparently added for the generic specialization feature and I don't think it's used in the access-control side of type-checking.